waddrmgr/common_test.go

```diff
@@ -53,6 +53,149 @@ var (
 
 	// waddrmgrNamespaceKey is the namespace key for the waddrmgr package.
 	waddrmgrNamespaceKey = []byte("waddrmgrNamespace")
+
+	// 	expectedExternalAddrs is the list of expected external addresses
+	// 	generated from the seed
+	expectedExternalAddrs = []expectedAddr{
```

Comment:
Make a map of expected values, remove them as they're found, error if any are left.